src/libs/OptionsListUtils.js

```diff
@@ -599,6 +599,32 @@ function isCurrentUser(userDetails) {
     return _.some(_.keys(loginList), (login) => login.toLowerCase() === userDetailsLogin.toLowerCase());
 }
 
+/**
+ * Calculates count of all enabled options
+ *
+ * @param {Object[]} options - an initial strings array
+ * @param {Boolean} options[].enabled - a flag to enable/disable option in a list
+ * @param {String} options[].name - a name of an option
+ * @returns {Boolean}
+ */
+function getEnabledCategoriesCount(options) {
+    return _.chain(options)
+        .filter((option) => Boolean(option.enabled))
```

Comment:
```suggestion
        .filter((option) => option.enabled)
```

If `option.enabled` is boolean we don't need to cast it.

Reply:
Yes, I agree. It looks strange)

Reply:
Done - 4f875ea1641b11fa676b5bafbd3c0f3facd2d1fc.